README.md

```diff
@@ -0,0 +1,17 @@
+# Flex Panels Image Gallery
+
+This is a simple static website which enables user to flex a gallery of images when click on it. 
+
+You can check it out live page from this [link](abelroland.github.io/flex-panel/).
+
+This project was created from the tutorials of [Wes Bos](https://github.com/wesbos/JavaScript30/tree/master/01%20-%20JavaScript%20Drum%20Kit) . We recommend you to watch his tutorials.
```

Comment:
Hey @razvanbrb , the link here is sending to the challenge N. 1 (Drum Kit). Maybe we should send the readers, or for the master folder of the project, or for the folder of the `flex panel` challenge 😉

Reply:
Ok, good that you notice. I will change it. :+1: